backend/src/Api/App/Controller/InitController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Api\App\Controller;
+
+use App\Entity\Publication;
+use App\Repository\ItemRepository;
+use App\Repository\PublicationRepository;
+use App\Repository\CollectionRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/init', name: 'app_api_init_')]
+class InitController extends AbstractController
+{
+    public function __construct(
+        private readonly ItemRepository $itemRepository,
+        private readonly PublicationRepository $publicationRepository,
+        private readonly CollectionRepository $collectionRepository,
+    ) {
+    }
+
+    #[Route('', name: 'default', methods: ['GET'])]
```

Comment:
as a convention, keep the full Route name near the method name. And, no need to explicitly set a 'name' for the route.

```
#[Route('/init', methods: 'GET')]
```